### PR TITLE
Removed account id to reduce confusion

### PR DESCRIPTION
**Pull request by Pectojin**
Cosmetic change for https://github.com/duplicati/duplicati/issues/3366#issuecomment-460830882

Nothing needs to change in the backend code, since it's always worked with Application ID, but since Account ID does not work anymore this will reduce confusion.

### CODE REVIEW

**Comment by Pectojin** [comment]
Yes, all languages will have to retranslate the phrase. Which is ideal since they've all been saying the wrong thing presumably.

**Comment by ts678** [comment]
[b2_authorize_account](https://www.backblaze.com/b2/docs/b2_authorize_account.html) says:

You can use either the master application key or a normal application key.

You'll find the master key on the B2 Cloud Storage Buckets page on the web site. When using the master key, the "key id" is your account ID, and the "application key" is the one you got from the web site.

and both ways still work here. I'm worried by [rclone](https://rclone.org/b2/) which sounds like yours, so I'm not sure which is correct...

**Comment by ikjadoon** [comment]
Thank you for the follow-up, ts678, and thank you for creating this commit, pectojin.

[My previous comment simplified](https://github.com/duplicati/duplicati/issues/3366#issuecomment-460830882) it a bit too much and ts678 is correct that they _both can work_, but the wording in Duplicati is just a tad off.

In B2, there are two types of access. To give access to **all buckets**, the API will only accept:

1. `Account ID` (automatically generated for each B2 account) plus...
2. `Master Application Key` (must be generated in the B2 Buckets menu)

To_ give access to **one bucket**, the API will only accept:

1. `Key ID` (generated upon creating a new application key) plus...
2. `Application Key` (generated upon creating a new application key)

Thus, _**four**_ unique IDs and only the "matching set" will authorize properly in the B2 API. Account ID & Master Application ID must always go together; Key ID and Application Key must always go together. I could see Duplicati users needing either or other, so perhaps removing "account ID" might end up adding more confusion. 

I apologize for my previous inaccurate comment.

For what it's worth, B2's documentation explains this distinction....not easily and I needed to read it thrice to understand the two authentication sets. For further validation, [Transmit's explanation matches the above](https://library.panic.com/transmit/transmit5/bb2-faq/), as does [this comment by a Backblaze employee](https://www.backblaze.com/blog/b2-application-keys/#comment-4015356187):

image

### TL;DR both can be used, but Duplicati should add "**Master** Application Key" to the text

Perhaps like so:

**Account ID** (all buckets) or **Key ID** (one bucket)
**Master Application Key** (all buckets) or **Application Key** (one bucket)

If that's too much text, then perhaps just adding "Master Application Key" to the second box's prompt and users can try to mentally match them.

**Comment by Pectojin** [comment]
That makes sense, although a bit confusing. 

I think "application id" does a decent job of conveying "[bucket or master] application id", since both are called application id's.

**Comment by ikjadoon** [comment]
Absolutely confusing, I agree. B2's vague GUI doesn't help. 

// 

Valid point, though B2 doesn't ever actually use "Application ID" and you can't find that on the interface. It's either "ApplicationKeyID" (API's name) or "KeyID" (interface's name).

image

Maybe `Key/Account ID` and `Application/Master Key` ?
